version_information_endpoint.asciidoc

```diff
@@ -176,27 +175,27 @@ List of known versions.
 |===
 |Value |Description 
 
-|2.0 |OCPI version 2.0. 
-|2.1 |OCPI version 2.1. (DEPRECATED, do not use, use 2.1.1 instead) 
-|2.1.1 |OCPI version 2.1.1.
-|2.2 |OCPI version 2.2. (this version)
+|2.0 |OCPI version 2.0
+|2.1 |OCPI version 2.1 (DEPRECATED, do not use, use 2.1.1 instead) 
+|2.1.1 |OCPI version 2.1.1
+|2.2 |OCPI version 2.2 (this version)
 |===
 
 [[version_information_endpoint_custom_modules]]
 ===== Custom Modules
 
 Parties are allowed to create custom modules or customized versions of the existing modules.
-For this the <<version_information_endpoint_moduleid_enum,ModuleID enum>> can be extended with additional custom moduleIDs.
+To do so, the <<version_information_endpoint_moduleid_enum,ModuleID enum>> can be extended with additional custom moduleIDs.
 These custom moduleIDs MAY only be sent to parties with which there is an agreement to use a custom module. Do NOT send custom moduleIDs to parties you are not 100% sure will understand the custom moduleIDs.
-It is advised to use a prefix (country_code + party_id) for any custom moduleID, this ensures that the moduleID will not be used for any future module of OCPI.
+It is advised to use a prefix (e.g. country-code + party-id) for any custom moduleID, this ensures that the moduleID will not be used for any future module of OCPI.
 
 For example:
 `nltnm-tokens`
 
 [[version_information_get_endpoint]]
 ==== GET
 
-Fetch information about the supported endpoints and their URLs for this version.
+Fetch information about the supported endpoints and their URLs for this OICP version.
```

Comment:
OICP? No, that is a different protocol: OCPI

Reply:
How embarrassing. 😅